desktop-app/package.json

```diff
@@ -40,24 +39,25 @@
     "test-e2e-live": "node -r @babel/register ./internals/scripts/CheckBuiltsExist.js && cross-env NODE_ENV=test testcafe-live electron:./ ./test/e2e/HomePage.e2e.js",
     "test-watch": "yarn test --watch"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
   "lint-staged": {
     "*.{js,jsx}": [
       "cross-env NODE_ENV=development eslint --cache --format=pretty",
-      "prettier --ignore-path .eslintignore --single-quote --write",
-      "git add"
```

Comment:
@ruisaraiva19 We don't need the `git add` any more?

Reply:
@manojVivek Yes, since `lint-staged` version `10.0.0`, it automatically includes `git add` as the final step.